EU4toV2/Source/V2World/Output/outRelation.cpp

```diff
@@ -1,15 +1,14 @@
-#include "outRelation.h"
+#include "output.h"
 
 std::ostream& V2::operator<<(std::ostream& output, const Relation& relation)
 {
-	output << "\t" << relation.target << "=\n";
-	output << "\t{\n";
-	output << "\t\tvalue=" << relation.value << "\n";
+	output << "" << relation.target << " = {\n";
```

Comment:
Can simplify to 

    output << relation.target << " = {\n";

Reply:
thanx, fixed.